inc/functions/cloudflare.php

```diff
@@ -72,7 +72,7 @@ function rocket_is_api_keys_valid_cloudflare( $cf_email, $cf_api_key, $cf_zone_i
 
 			if ( ! empty( $cf_zone->result ) ) {
 				$parsed_url = wp_parse_url( $site_url );
-				if ( strtolower( $parsed_url['host'] ) === $cf_zone->result->name ) {
+				if ( strpos( strtolower( $parsed_url['host'] ), $cf_zone->result->name ) >= 0 ) {
```

Comment:
```suggestion
				if ( false !== strpos( strtolower( $parsed_url['host'] ), $cf_zone->result->name ) ) {
```

We usually do the comparison with `strpos` like this, I'm not sure there is any actual benefit, but for the sake of consistency with the rest of the code let's do it the same everywhere :)